src/excalidraw-app/data/firebase.ts

```diff
@@ -106,7 +110,13 @@ const encryptElements = async (
   key: string,
   elements: readonly ExcalidrawElement[],
 ): Promise<{ ciphertext: ArrayBuffer; iv: Uint8Array }> => {
-  const json = JSON.stringify(elements);
+  const json = JSON.stringify(
+    elements.filter(
+      (el) =>
+        !el.isDeleted ||
+        el.updated >= getUpdatedTimestamp() - DELETED_ELEMENT_TIMEOUT,
```

Comment:
```suggestion
// Drop deleted elements older than a day
        el.updated > getUpdatedTimestamp() - DELETED_ELEMENT_TIMEOUT,
```
lets start with deleted elements older than a day

Reply:
Also done in f6f4da1.